components/Buttons/src/MDCButton.m

```diff
@@ -667,7 +667,7 @@ - (void)setElevation:(CGFloat)elevation forState:(UIControlState)state {
 #pragma mark - Border Color
 
 - (UIColor *)borderColorForState:(UIControlState)state {
-  return _borderColors[@(state)];
+  return _borderColors[@(state)] ?: _borderColors[@(UIControlStateNormal)];
 }
 
 - (void)setBorderColor:(UIColor *)borderColor forState:(UIControlState)state {
```

Comment:
Where are the changes to `setBorderColor`? Please see related methods like `setBorderWidth:forState:` and https://github.com/material-components/material-components-ios/pull/5919 for examples and context about how these properties should work.  There are complexities around `UIControlStateHighlighted` and `UIControlStateDisabled` that need to be accounted for.

Reply:
Addressed in 9edc2664fa337b4dcfc963d30246ee5c44492768